addons/hr_expense/models/hr_expense.py

```diff
@@ -848,6 +861,7 @@ def _default_bank_journal_id(self):
 
     name = fields.Char('Expense Report Summary', required=True, tracking=True)
     expense_line_ids = fields.One2many('hr.expense', 'sheet_id', string='Expense Lines', copy=False)
+    product_ids = fields.Many2many('product.product', compute='_compute_product_ids', store=True, string='Categories', domain="[('can_be_expensed', '=', True), '|', ('company_id', '=', False), ('company_id', '=', company_id)]")
```

Comment:
The domain is not needed as the field is not editable :innocent: 

```suggestion
    product_ids = fields.Many2many('product.product', compute='_compute_product_ids', store=True, string='Categories')
```